nextcloud_install_production.sh

```diff
@@ -395,7 +395,7 @@ occ_command config:system:set upgrade.disable-web --value="true"
 print_text_in_color "$ICyan" "Configuring update notifications specific for this server..."
 download_static_script updatenotification
 check_command chmod +x "$SCRIPTS"/updatenotification.sh
-crontab -u root -l | { cat; echo "59 $AUT_UPDATES_TIME * * 6 $SCRIPTS/updatenotification.sh > /dev/null 2>&1"; } | crontab -u root -
+crontab -u root -l | { cat; echo "59 $AUT_UPDATES_TIME * * * $SCRIPTS/updatenotification.sh > /dev/null 2>&1"; } | crontab -u root -
```

Comment:
Okay, that makes sense 👍
But what is a good time when we run it every day?

Reply:
I actually don't think we should send notifications more often than once per week.

Reply:
Yes, we are not sending a notification every day. I've managed to do the following with this PR:
If the notification about a new software update was sent once, no notification will be sent until the next update is available. But we check for it every day. And are able to send a notification to the user as soon as the update becomes available.

Reply:
@enoch85 You can test if it works by running the script and add manually below REPORTEDNCVERSION:
`NCVERSION=18.0.2`
when you 
run the script now, it will report, that 18.0.2 is available. But if you run it a second time it will send no notification because the varaiable REPORTEDNCVERSION will be set to 18.0.2
If you then set NCVERSION e.g. to 18.0.3 it will report for the user once that 18.0.3 is available but only this one time because it will set REPORTEDNCVERSION to 18.0.3... and so on

Reply:
I've already tested it by myself and it works!